app/assets/stylesheets/_layout.scss

```diff
@@ -120,6 +122,84 @@ header {
   }
 }
 
+.samples-dropdown {
+  
+  .samples-dropdown-text {
+    cursor: pointer;
+    padding-left: 20px;
+    color: $gray-light;
+    position: relative;
```

Comment:
`position: relative` needs to go on the element that's enclosing the dropdown menu. With this we can get rid of the JavaScript setting the left margin.

```suggestion
.samples-dropdown {
  position: relative;
  
  .samples-dropdown-text {
    cursor: pointer;
    padding-left: 20px;
    color: $gray-light;
```